pipelines/nextflow/modules/gff3/process_gff3.nf

```diff
@@ -13,23 +13,24 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-params.merge_split_genes="False"
 
 process PROCESS_GFF3 {
     tag "${meta.accession}"
     label 'adaptive'
+    label 'cached'
```

Comment:
```suggestion
```
This should not be cached: if we make changes to the process (e.g. adding acceptable biotypes), then we need to have it rerun, but without redownloading the data from genbank).